pentaho-webpackage/extenders/http-extender/src/main/java/org/pentaho/webpackage/extender/http/impl/PentahoWebPackageServiceImpl.java

```diff
@@ -31,7 +35,7 @@
  * Implementation of the WebContainer service.
  */
 public class PentahoWebPackageServiceImpl implements PentahoWebPackageService, BundleListener {
-  private final Map<Long, PentahoWebPackageBundle> pentahoWebPackageBundles = new HashMap<>();
+  private final Map<Long, PentahoWebPackageBundleImpl> pentahoWebPackageBundles = new HashMap<>();
```

Comment:
@nantunes switch for synchronized hash map. ConcurrentHashMap seems like an overkill for the case.